### PR TITLE
 Update zh-rTW translation for CrAzYpass 

### PR DESCRIPTION
**Pull request by Lafudoci**
Update translations for CrAzYpass. The about.xml will be updated later.

### CODE REVIEW

**Comment by erciccione** [comment]
Thanks @Lafudoci . I updated Taiga: https://taiga.getmonero.org/project/erciccione-monero-localization/us/220?milestone=28

File: app/src/main/res/values-zh-rTW/help.xml [hunk 1]
```diff
@@ -56,26 +56,21 @@
...
-        <h3>CrAzYpass wallets</h3>
-        If you ever need to reinstall Monerujo (for example after resetting your phone or switching
-        to a new one) or you want to use your wallet files on a different device or PC, you have to
-        use this Recovery Password in order to access your wallet again.<br/>
-        By selecting \"Change Passphrase\" from the menu, you can choose another passphrase. Beware
-        that this will generate a new Recovery Password. Write it down!
+        <h2>錢包檔回復密碼</h2>
+        請務必抄寫下此份密碼。當你重置手機或重新安裝APP時，你將會需要用它來重新存取錢包檔。<br/>
+        <h3>CrAzYpass加密</h3>
+        若你的密碼顯示為4組的52英數字元，恭喜你!
```
Comment (m2049r) [inline app/src/main/res/values-zh-rTW/help.xml]: there is an extra blank at the end of this line.

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/help.xml]: OK, fixed.

**Review by Pmking3203 — APPROVED** [review]

**Review by Pmking3203 — APPROVED** [review]
It's good.

**Review by m2049r — APPROVED** [review]

**Pull request merged**